Source/Orts.Parsers.Msts/STFReader.cs

```diff
@@ -1170,6 +1180,23 @@ internal double ParseUnitSuffix(ref string constant, UNITS validUnits)
                     case "wh": return 3.6e3f;
                     case "kwh": return 3.6e6f;
                 }
+            if ((validUnits & UNITS.ForceRate) > 0)
+                switch (suffix)
+                {
+                    case "": return 1.0;
+                    case "n/s": return 1;
+                    case "kn/s": return 1e3;
+                    case "lbf/s": return 4.44822162;
+                    case "lb/s": return 4.44822162;
+                }
+            if ((validUnits & UNITS.PowerRate) > 0)
```

Comment:
I did more experimenting with this latest version and found I couldn't get power rates to work...Looks like the limitations of the units enum are starting to bite, as `(validUnits & UNITS.PowerRate)` returns `-2147483648` when validUnits is UNITS.PowerRate, seems that gets casted to a signed value before doing the comparison. I think for now you can get away with changing the comparison to != 0, or casting the result of the bitwise operation to a (uint).
```suggestion
            if ((validUnits & UNITS.PowerRate) != 0)
```

Reply:
Yes, I thought that 1<<31 was moving to the 31th bit but it's actually moving to the 32nd, as 1<<0 already uses the first bit. I changed the enum to be ulong-based instead of int-based to temporary increase the number of different units to 63. Hope that works. Meanwhile, I will think about how to change the enum so we can still use Compulsory and Any without having to use one bit per unit, but that will go in a separate PR.